bin/commands/internal/config/get/index.ts

```diff
@@ -21,9 +22,26 @@ export function execute(configPath:string, haInstance?: string) {
   common.requireZoweYaml();
   const ZOWE_CONFIG=config.getZoweConfig();
   let output;
-  if (haInstance) {
-    haInstance=config.sanitizeHaInstanceId();
+  const spaces = std.getenv('ZWE_CLI_PARAMETER_FORMAT') == 'true' ? 2 : 0;
+  
+  // It is possible to define HA instance in mixed case, so we'll search for:
+  //   - original instance ID
+  //   - sanitized ID
+  //   - 1st lowercased match
+  if (haInstance && ZOWE_CONFIG.haInstances) {
+    if (!ZOWE_CONFIG.haInstances[`${haInstance}`]) {
+      haInstance = config.sanitizeHaInstanceId();
```

Comment:
previously, this code always sanitized the HA instance ID, and the sanitize function modifies some environment variables.  

If we don't hit the sanitize function in these conditionals, will any downstream code be impacted by skipping the environment variable updates? Should the env updates be separated from `sanitizeHaInstanceId`,  and always called?

Reply:
I don't think the environment update was ever needed here. The `sanitizeHaInstanceId` is used as "string" function. However there is a similar code in `zwe internal config set`. I will revert this part and open issue to be discussed by others.

Let's keep this PR simple.